mllib/src/main/scala/org/apache/spark/mllib/clustering/PowerIterationClustering.scala

```diff
@@ -417,6 +435,17 @@ object PowerIterationClustering extends Logging {
       .setK(k)
       .setSeed(0L)
       .run(points.values)
-    points.mapValues(p => model.predict(p)).cache()
+
+    val predict = points.mapValues(p => model.predict(p)).cache()
```

Comment:
Can just be `.mapValues(model.predict)` while we're here. Same with `Vectors.dense` above.

Reply:
modified to `.mapValues(model.predict(_))` also for Vectors.dense